### PR TITLE
feat: landing page for info session

### PR DESCRIPTION
**Pull request by Sean1572**

### CODE REVIEW

**Comment by ntlhui** [comment]
image

QR Code: 
image

**Comment by ntlhui** [comment]
Does it make sense to link the emails?  `[nthui@ucsd.edu](mailto:nthui@ucsd.edu)`

File: ayl.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- Benjamin McCalmon, bbmcca26@g.holycross.edu
+
+## [FishSense End User]({{ "fishsense" | relative_url }})
+- Harish Vasanth, havasanth@ucsd.edu
+
+## [Smartfin]({{ "smartfin" | relative_url }})
+- Nathan Hui, nthui@ucsd.edu
+
+## [Radio Telemetry Tracking]({{ "radio-collar-tracker" | relative_url }})
+- Tyler Flar (Radio Telemetry Tracking) – tflar@ucsd.edu
```
Comment (ntlhui) [inline ayl.md:26]: Inconsistent formatting

File: ayl.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Sean Perry, shperry@ucsd.edu
+- Ludwig von Schoenfeldt, lvonschoenfeldt@ucsd.edu
+- TQ Zhang, tiz019@ucsd.edu 
+
+##  [Mangrove Monitoring]({{ "mangrove-monitoring" | relative_url }}) 
+- Gage Wyre, gwrye100@gmail.com 
+- Zach Lawrence, zclawrence@ucsd.edu
+
+## [Hardware Dev/Research Support]({{ "research-support-group" | relative_url }})   
+- Nathan Hui, nthui@ucsd.edu
```
Comment (ntlhui) [inline ayl.md:38]: Let's drop this, I've removed it as an option from the application, and I don't really have time to manage it.

File: ayl.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+## [FishSense End User]({{ "fishsense" | relative_url }})
+- Harish Vasanth, havasanth@ucsd.edu
+
+## [Smartfin]({{ "smartfin" | relative_url }})
+- Nathan Hui, nthui@ucsd.edu
+
+## [Radio Telemetry Tracking]({{ "radio-collar-tracker" | relative_url }})
+- Tyler Flar (Radio Telemetry Tracking) – tflar@ucsd.edu
+
+## [Radio Telemetry Tracking]({{ "acoustic-species-identification" | relative_url }})
```
Comment (ntlhui) [inline ayl.md:28]: Wrong text

**Review by ntlhui — CHANGES_REQUESTED** [review]

**Comment by ccrutchf** [comment]
Do we want to link the papers here?

**Comment by Sean1572** [comment]
I've linked project webpages, should we add a link to onboarding papers?

**Comment by Sean1572** [comment]
> image
> 
> QR Code: image

check qr code, its ayl not qyl

**Comment by Sean1572** [comment]
image

**Review by TQZhang04 — APPROVED** [review]
LGTM - Sean

**Review by ntlhui — APPROVED** [review]

**Pull request merged**